web/libs/editor/src/utils/canvas.js

```diff
@@ -352,12 +352,22 @@ function Region2RLE(region) {
 function brushSizeCircle(size) {
   const canvas = document.createElement('canvas');
   const ctx = canvas.getContext('2d');
+  const canvasSize = size * 4 + 8;
+  const circlePos = size / 2 + 4;
```

Comment:
I see this is copy pasted from before, but what is 4? what is 8? can we get those defined as constants?